libs/feed/src/components/beam-content-resolver.tsx

```diff
@@ -13,7 +13,7 @@ export type BeamContentResolverProps = {
 
 const BeamContentResolver: React.FC<BeamContentResolverProps> = ({
   beamId,
-  showNSFWCard,
+  showNSFWCard = false,
```

Comment:
Can you explain a bit how is this flag solving the cards click and why isn't correctly passed from the parent?